src/CommonLib/Processors/LDAPPropertyProcessor.cs

```diff
@@ -705,6 +766,97 @@ private enum IsTextUnicodeFlags
         }
     }
 
+    /// <summary>
+    /// Provides single-truth mapping of domain object properties.
+    /// </summary>
+    public static class PropertyMap
+    {
+        /// <summary>
+        /// Get output name of a domain object property.
+        /// </summary>
+        /// <param name="ldapProperty"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentException"></exception>
+        public static string GetPropertyName(string ldapProperty)
+        {
```

Comment:
I'm not certain this will live here or that it should take this form as a method instead of a static class providing static properties, but this provides a glossary of LDAP properties and our own internal names for them.